lib/fastlane/actions/pubu.rb

```diff
@@ -0,0 +1,70 @@
+module Fastlane
+  module Actions
+    class PubuAction < Action
+      def self.run(params)
+        url = ENV['PUBU_URL']
+        message = params[:message].to_s
+        pubuayload = build_payload(message, params)
+        send_pubu(url, pubuayload)
+        raise "`PUBU_URL` lost  please set ENV first`".red unless url
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :message,
+                                       description: "The message will displayed on PubuIM",
+                                       optional: true),
+
+          FastlaneCore::ConfigItem.new(key: :pubuUrl,
+                                       env_name: "PUBU_URL",
+                                       description: "PubuIM services url", #todo add doc url here
+                                       optional: true,
+                                       verify_block: proc do |value|
+                                           raise "Invalid PUBU_URL" unless value.start_with? "https://hooks.pubu.im/services/"
+                                         end),
```

Comment:
`end` at 24, 41 is not aligned with `proc do |value|` at 22, 53 or `verify_block: proc do |value|` at 22, 39.